lib/ui/semantics.dart

```diff
@@ -527,6 +529,27 @@ class SemanticsFlag {
   /// navigate to the next page when reaching the end of the current one.
   static const SemanticsFlag hasImplicitScrolling = SemanticsFlag._(_kHasImplicitScrollingIndex, 'hasImplicitScrolling');
 
+  /// The semantics node has the quality of either being "expanded" or "collapsed".
+  ///
+  /// For example, a submenu button widget has expanded state.
```

Comment:
```suggestion
  /// For example, a [SubmenuButton] widget has expanded state.
```

Reply:
not entirely sure if the doc string can pick this up.

Reply:
Just checked other [api doc](https://api.flutter.dev/flutter/dart-ui/SemanticsFlag/hasImplicitScrolling-constant.html) and looks like using square bracket can link to the widget doc. Thanks for the suggestion:)